README.md

```diff
@@ -120,10 +95,6 @@ For more usage examples, check out [the tests](https://github.com/googlemaps/goo
 Automatically retry when intermittent failures occur. That is, when any of the retriable 5xx errors
 are returned from the API.
 
-### Client IDs
```

Comment:
Revert this? In Java we still have it referenced: https://github.com/googlemaps/google-maps-services-java/pull/659/files#diff-04c6e90faac2675aa89e2176d2eec7d8R191 (or remove on both repos)

Reply:
Now removed in both.